TodoListPosta/app/components/input/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import { TextInput } from 'react-native'
+
+import styles from './styles';
+
+export default class Input extends Component {
```

Comment:
Remember to put `export default` at the bottom of this file. This way, it becomes a bit more comfortable when adding HOCs